### DISCUSSION TITLE
How Long Does a Download Live in Metadata?

### DISCUSSION THREAD

**Discussion by Budlyte** [advplyr/audiobookshelf#86] [General]
If I queue up a download on PC, I tested as a zip, but then close the window and abandon the zip it stays in the metadata/downloads folder.  How long will it sit there?

Also, I did this because the mobile app isn't able to download anymore. I just downloaded a couple last night and then tried again today after updating the server to v1.4.0.  Also tried updating the app and still no bueno.

**Comment by advplyr** [comment]
The download gets deleted after an hour, or if you restart the server it will be removed.

I don't remember the download issue you were having, what is it saying/doing?
